decidim-core/app/packs/src/decidim/index.js

```diff
@@ -5,10 +5,10 @@ import formDatePicker from "src/decidim/form_datepicker"
 import fixDropdownMenus from "src/decidim/dropdowns_menus"
 import createQuillEditor from "src/decidim/editor"
 import Configuration from "src/decidim/configuration"
-import ExternalLink from "src/decidim/redesigned_external_link"
+import ExternalLink from "src/decidim/external_link"
```

Comment:
Is it on purpose you are reverting this and the character counter back to original?

Reply:
Yes, because the index.js is the legacy one. The new one in redesigned_index.js